mitodl/micromasters
# upgrade redux-hammock

#### Pre-Flight checklist


#### What are the relevant tickets?

none

#### What's this PR do?

I recently upgraded some dependencies in redux-hammock (mitodl/redux-hammock#31). This just bumps us up to that version here.

#### How should this be manually tested?

We don't use redux-hammock for everything in MM, but we do use it for these reducers:

```
  automaticEmailsEndpoint,
  courseEnrollmentsEndpoint,
  coursePricesEndpoint,
  programLearnersEndpoint,
  discussionsFrontpageEndpoint,
  channelsEndpoint
```

so I think we should just check that the functionality for all of those things is working correctly.
